crates/routes/src/feeds.rs

```diff
@@ -68,32 +70,39 @@ async fn get_all_feed(
   info: web::Query<Params>,
   context: web::Data<LemmyContext>,
 ) -> Result<HttpResponse, Error> {
-  let sort_type = get_sort_type(info).map_err(ErrorBadRequest)?;
-  Ok(get_feed_data(&context, ListingType::All, sort_type).await?)
+  let sort_type = get_sort_type(&info).map_err(ErrorBadRequest)?;
+  let limit = info.limit.unwrap_or(RSS_FETCH_LIMIT);
+  let page = info.page.unwrap_or(1);
+  Ok(get_feed_data(&context, ListingType::All, sort_type, limit, page).await?)
```

Comment:
Would be cleaner if you move this stuff into a helper function in `impl Params`.

Reply:
Good idea - pushed this change